programs/programs/il/pe/member.py

```
@@ -62,3 +65,61 @@ class IlAabd(PolicyEngineMembersCalculator):
+        # income eligible
+        # not providing il_aabd_expense_exemption_person - includes details we don't have
+        # conservative estimate by excluding it (since subtracted from income)
+        member_dependency.IlAabdGrossEarnedIncomeDependency,
+        member_dependency.IlAabdGrossUnearnedIncomeDependency,
+        member_dependency.IlHbwdCountableUnearnedIncomeDependency,
+        # asset eligibility
+        # not including il_aabd_countable_vehicle_value since we don't have
+        spm_dependency.CashAssetsDependency,
+        # state requirement
+        dependency.IlStateCodeDependency,
+    ]
```

Ruff (0.14.8)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)